tachyon/math/elliptic_curves/msm/algorithms/icicle/icicle_msm.h

```diff
@@ -17,6 +20,78 @@
 
 namespace tachyon::math {
 
+namespace {
+
+// NOTE(GideokKim): The formula for memory usage estimation provided in the
+// document did not match the actual memory allocation, so some of the formula
+// was modified. |scalars_memory_size| and |points_memory_size| are exactly the
+// same, and |scalar_indices_memory_size| internally uses the sort function of
+// the cub library to set some free memory. In the case of |bucket_memory_size|,
+// it uses more memory than the actual formula, so it was modified to an
```

Comment:
bucket -> buckets, & change the wording slightly.
```suggestion
// the cub library to set some free memory. |buckets_memory_size| uses more memory than the actual formula, so it was modified to an...